docs/API.md

```diff
@@ -3,47 +3,59 @@
 The current live api lives at https://api.jujucharms.com/charmstore/v4
 
 ## Intro
-The charm store stores and indexes charms and bundles. A charm or bundle is referred to by a charm store id which can take one of the following two forms:
+
+The charm store stores and indexes charms and bundles. A charm or bundle is
+referred to by a charm store id which can take one of the following two forms:
 
 * ~owner/series/name(-revision)
```

Comment:
can we italicise "owner", "series", "name" and "revision here and in the line below, please?